comparators/priority_area.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+var fs = require('fs');
+var path = require('path');
+var turfWithin = require('turf-within');
+var turfCentroid = require('turf-centroid');
+var turfFC = require('turf-featurecollection');
+
+module.exports = priority_area;
+
+/**
+* Returns a flag if a feature is located in a priority area
+* @param {object} newVersion Features new version in GeoJSON.
+* @param {object} oldVersion Features old version in GeoJSON.
+* @param {Function} callback called with (error, result).
+* @returns {undefined} calls callback.
+*/
+function priority_area(newVersion, oldVersion, callback) {
+
```

Comment:
@planemad Since this compare function needs the `newVersion` of the feature to have a geometry, we could put a small check like below before the calculations.
```
if (!newVersion || !newVersion.geometry) return callback(null, {});
```